Formula/qt5-webkit.rb

```diff
@@ -262,7 +264,7 @@ class Client : public QObject
          PORT=Qt \
          CMAKE_BUILD_TYPE=$$configuration \
          CMAKE_TOOLCHAIN_FILE=$$toolchain_file \
-+        CMAKE_INSTALL_PREFIX=/usr/local/Cellar/qt5-webkit/5.11.2
++        CMAKE_INSTALL_PREFIX=HOMEBREW_PREFIX/Cellar/qt5-webkit/5.12.0
```

Comment:
I believe `HOMEBREW_PREFIX`will be read as a string and not a variable.

Reply:
Tested, works correctly!